output.php

```diff
@@ -1,12 +1,26 @@
 <?php
-
+$sampleSize = 5;
+$threshold = 250;
 $db = new SQLite3("buerostatus.db");
-$result = $db->query("SELECT val FROM buerostatus ORDER BY id DESC LIMIT 1;");
-$val = $result->fetchArray()["val"];
-if ($val > 250) {
-	echo 1;
-} else {
-	echo 0;
+$result = $db->query("SELECT val FROM buerostatus ORDER BY id DESC LIMIT $sampleSize;");
+$row = $result->fetchArray();
+
+$vals = [];
+while ($row != FALSE && count($row)>0) {
+        $vals[] = $row['val'];
+        $row = $result->fetchArray();
 }
 
+$medianElement = floor($sampleSize/2);
+sort($vals, SORT_NUMERIC);
+$val = $vals[$medianElement];
+if(isset($_GET["raw"])) {
+        echo $val;
```

Comment:
Ich hab nur gemacht, was in #2 stand – letzter gemessener Wert lässt sich natürlich auch ausgeben bei Bedarf, könnte man dann über ?raw und ?median machen oder so